__tests__/app.test.js

```diff
@@ -94,6 +94,80 @@ describe("/api/articles", () => {
         expect(articles).toEqual(sortedArticles);
       });
   });
+  test("GET 200: responds with an array of article objects in descending order", () => {
+    return request(app)
+      .get("/api/articles?order=desc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        const sortedArticles = [...articles].sort(
+          (a, b) => b.created_at - a.created_at
+        );
+        expect(articles).toEqual(sortedArticles);
+      });
+  });
+  test("GET 200: responds with an array of article objects in ascending order", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        const sortedArticles = [...articles].sort(
+          (a, b) => a.created_at - b.created_at
+        );
+        expect(articles).toEqual(sortedArticles);
+      });
+  });
+  test("GET 200: responds with an array of article objects sorted by given column", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        const sortedArticles = [...articles].sort(
+          (a, b) => a.created_at - b.created_at
```

Comment:
We are asserting that we are sorting by votes but your sort is ordering by created_at?